<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAdminWin</class>
 <widget class="QDialog" name="MainAdminWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>791</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="WelcomeLabel">
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000ff;&quot;&gt;Welcome &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WelcomeLabel_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(255, 170, 255);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Role: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ServerStateLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Server isn't connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="PB_GetAccountNumber">
      <property name="text">
       <string>Get account number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewAccountBalance">
      <property name="text">
       <string>View accountBalance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewTransactionHistory">
      <property name="text">
       <string>View transaction history</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_ViewBankDataBase">
      <property name="text">
       <string>View Bank DataBase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_CreateNewUser">
      <property name="text">
       <string>Create New User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_DeleteUser">
      <property name="text">
       <string>Delete User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PB_UpdateUser">
      <property name="text">
       <string>Update User</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>7</number>
   </property>
   <widget class="QWidget" name="EmtyPage"/>
   <widget class="QWidget" name="GetAccountNumber">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Enter the user name: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_GetAccountNumber"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="PB_GetAccounNumber">
          <property name="text">
           <string>Get Account Number</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="AccountNumberLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Enter the Account number: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_GetAccountBalance"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="PB_GetAccounBalance">
          <property name="text">
           <string>Get Account Balance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="BalanceLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="TV_ViewTransactionHistory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Enter the Account number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_ViewTransactionHistory"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_View">
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="TV_ViewBankDataBase">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4"/>
   <widget class="QWidget" name="page_5">
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Enter the Account number: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_DeleteUserID"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Enter the Account signature: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LE_DeleteUserSignature"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_DeleteUserSignature">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DeleteUserLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>206</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Enter the Account number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateUserID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>What do you want to change: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="UpdateComboBox">
          <item>
           <property name="text">
            <string>Account number</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account User name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account National ID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account Type</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account interset rate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Account type</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Enter the new value: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateNewValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Enter the account signature: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_UpdateSignature"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="UpdateLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
